### PR TITLE
Add --quiet flag

### PR DESCRIPTION
**Pull request by mgrachev**
Waiting for dotenv-linter [v2.2.0](https://github.com/dotenv-linter/dotenv-linter/issues/238) release.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 cd "$GITHUB_WORKSPACE"
 export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
-dotenv-linter ${INPUT_DOTENV_LINTER_FLAGS} \
+dotenv-linter --quiet ${INPUT_DOTENV_LINTER_FLAGS} \
```
Comment (DDtKey) [inline entrypoint.sh:6]: What about using the default input via `action.yml`? :thinking:

**Review by DDtKey — COMMENTED** [review]
Thank you for this update before next release!

File: entrypoint.sh (hunk 1 above)
Comment (mstruebing) [reply entrypoint.sh:6]: I also think that having it configurable would be better, is there a reason you've added that flag hardcoded?

Comment (mgrachev) [reply entrypoint.sh:6]: We should pass `reviewdog` only warnings without additional information, otherwise `reviewdog` won't work.

**Review by DDtKey — APPROVED** [review]

**Pull request merged**